### DISCUSSION TITLE
Texture mapping

### DISCUSSION THREAD

**Discussion by Warrenayiss** [bnpr/Malt#586] [Q&A] [answered]
I'm a beginner both in 3d and in Malt engine and I am learning about visual effects with Eevee while trying to replicate the workflow with Malt I faced an issue, the Mapping node I found in the vector category doesn't behave like the Mapping node in Eevee. How do I get the same control on noise texture and other types of textures in Malt as in Eevee?
![image](https://github.com/user-attachments/assets/d5eaf3b3-457e-48ab-8c1f-0e168af1e6e2)
The goal is to be at least able to scale the textures on one axis, have distortions in wave textures, or change the orientation of the waves

**Comment by pragma37** [comment, marked as answer]
> the Mapping node I found in the vector category doesn't behave like the Mapping node in Eevee

It should, in theory.

![imagen](https://github.com/user-attachments/assets/3cb92d4f-6304-402e-ad5d-e60df456b806)

If you have an example where it doesn't I can take a look.

**Reply by pragma37** [reply]
Well, there's one difference. EEVEE nodes use degrees, but Malt nodes use radians.
360 degrees is the same as `2*pi` radians, so you can convert degrees to radians by multiplying them by `pi/180`.
